billing-ui/package.json

```diff
@@ -40,14 +40,14 @@
         "eslint-plugin-react-hooks": "^4.1.0",
         "eslint-plugin-sort-destructure-keys": "^1.3.5",
         "file-loader": "6.2.0",
-        "filesize": "3.3.0",
-        "fs-extra": "0.30.0",
-        "gzip-size": "3.0.0",
+        "filesize": "6.1.0",
+        "fs-extra": "9.0.1",
+        "gzip-size": "5.1.1",
         "html-webpack-plugin": "^4.4.1",
         "json-loader": "0.5.7",
-        "mini-css-extract-plugin": "1.3.0",
+        "mini-css-extract-plugin": "1.3.1",
         "node-sass": "^5.0.0",
-        "opn": "4.0.2",
+        "opn": "6.0.0",
```

Comment:
deprecated package https://www.npmjs.com/package/opn

Reply:
OOH nice! let me switch that, see how it works